Clarification of contribution guidelines (#675)
corona-warn-app/cwa-app-android: CONTRIBUTING.md

All members of the project community must abide by the [Contributor Covenant, version 2.0](CODE_OF_CONDUCT.md).
Only by respecting each other we can develop a productive, collaborative community.
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting [corona-warn-app.opensource@sap.com](mailto:corona-warn-app.opensource@sap.com) and/or a project maintainer.

We appreciate your courtesy of avoiding political questions here. Issues which are not related to the project itself will be closed by our community managers.

## Engaging in Our Project

We use GitHub to manage reviews of pull requests.

* If you are a new contributor, see: [Steps to Contribute](#steps-to-contribute)

> * Before opening a new Pull Request, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue.

Reviewer: Something in the logic here is backward.

If you want people to open an issue before doing any work, then say so.
```suggestion
* Before implementing your change, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue.
```
or
```suggestion
* Before opening a new Pull Request, create an issue that describes the problem you would like to solve or the code that should be enhanced. Please note that you are willing to work on that issue. If you have already started, you can link to your fork.
```
At the point at which people can/could/would open a pull request, they have an implementation. If they have an implementation, they might as well link to their fork, as seeing the practical implementation might swing in favor or against a certain idea.

* When creating an issue, try using one of our issue templates which already contain some guidelines on which content is expected to process the issue most efficiently. If no template applies, you can of course also create an issue from scratch.

> * Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-android/labels) to your issue so that all community members are able to cluster the issues better.

Reviewer: Are you aware that only project members can add and remove labels on github?
Asking general github users/"the community" to apply labels is therefor a bit pointless.

Author: Thanks for spotting that. We'll check this sentence in the other repos, too.
